src/java/org/apache/cassandra/io/util/DataInputPlus.java

```diff
@@ -30,11 +30,31 @@
 @Shared(scope = SIMULATION)
 public interface DataInputPlus extends DataInput
 {
+    /**
+     * Read a 64-bit integer back. This assumes it was originally written using writeVInt
+     * and uses zigzag encoding.
+     * @return
```

Comment:
nit: Could remove `@return`...or potentially move the description into it? Maybe something like...

```
@return a 64-bit integer, assumed to originally have been written using {@link DataOutputPlus#writeVInt(long)}
```

Reply:
Removed the return..